build-manual.py

```diff
@@ -73,6 +73,73 @@ def __init__(self, number: str, title: str, filename: str):
 		self.filename = filename
 		self.items = []
 
+def make_one_page(dest_directory):
+	"""
+	Generate one-page php file of the manual.
+	"""
+	# Get all php files in destination directory.
+	php_files = list(filter(lambda x: x.endswith("php"), os.listdir(dest_directory)))
+	index = php_files.pop(php_files.index("index.php"))
+	dest_directory = Path(dest_directory)
+
+	with open(dest_directory / index, "r", encoding="utf-8") as file:
+		index_soup = BeautifulSoup(file, features="html.parser")
+		php_tags = regex.findall(r"<\?.+?\?>", index_soup.prettify(formatter=None), regex.S)
```

Comment:
One last thing, can you use the full name of the flag for clarity?

Reply:
@acabal  yup, changed to regex.DOTALL now